Source/SourceKittenFramework/SwiftDeclarationAttributeKind.swift

```diff
@@ -71,7 +71,7 @@ public enum SwiftDeclarationAttributeKind: String, CaseIterable {
     case setterPublic = "source.decl.attribute.setter_access.public"
     case setterOpen = "source.decl.attribute.setter_access.open"
     case optimize = "source.decl.attribute._optimize"
-    case consuming = "source.decl.attribute.__consuming"
+    case __consuming = "source.decl.attribute.__consuming"
```

Comment:
In conjunction with the new 'real' `consuming` attribute this rename is a bit dodgy, open to suggestions...